Pod/Classes/Deselection/UIViewController+Deselection.swift

```diff
@@ -30,64 +30,58 @@
 
 import UIKit
 
+public protocol SmoothlyDeselectableItems {
+    var indexPathsForSelectedItems: [IndexPath]? { get }
```

Comment:
Had to be because you are passing it into the `smoothDeselectItems(_ deselectable: SmoothlyDeselectableItems?)`